### ISSUE TITLE
google_compute_instance_group_manager update_strategy not properly read

### ISSUE THREAD

**Issue by JDiPierro** [hashicorp/terraform#10174]
### Terraform Version
`0.7.11`

### Affected Resource(s)
- google_compute_instance_group_manager

### Terraform Configuration Files
```hcl
resource "google_compute_instance_template" "test" {
  name_prefix = "test-"
  machine_type = "f1-micro"
  can_ip_forward = false
  region = "us-central1"

  disk {
    source_image = "centos-7"
  }

  network_interface {
    access_config {
      // Ephemeral IP
    }
  }

  lifecycle {
    create_before_destroy = true
  }

  service_account {
    email = "compute@project.iam.gserviceaccount.com"
    scopes = ["cloud-platform"]
  }
}

resource "google_compute_instance_group_manager" "test" {
  name = "test"
  instance_template = "${google_compute_instance_template.test.self_link}"
  update_strategy = "NONE"
  base_instance_name = "test"
  zone = "us-central1-b"
  target_size = "1"
}
```

### Expected Behavior
The instance group manager persists the `"NONE"` setting into the `tfstate`.

### Actual Behavior
"RESTART" is saved into the `tfstate`. All subsequent plans will show a pending change:

    ~ google_compute_instance_group_manager.test
          update_strategy: "RESTART" => "NONE"

### References

Looks like [this line](https://github.com/hashicorp/terraform/blob/master/builtin/providers/google/resource_compute_instance_group_manager.go#L245) is causing the problem. After update we go to read and read defaults `update_strategy` to `"REFRESH"`. I think this should be attempting to pull the `update_strategy` using `d.Get(...)` and set it to the default if `nil` is returned.

**Comment by evandbrown** [comment]
LGTM, thanks @paddyforan!

**Comment by paddyforan** [comment]
Het @evandbrown, is that for #11013? Just want to be sure before I make assumptions. :P

**Comment by stack72** [comment]
Closed via #11013
